lib/article_json/export/apple_news/elements/text.rb

```diff
@@ -0,0 +1,16 @@
+module ArticleJSON
+  module Export
+    module AppleNews
+      module Elements
+        class Text < Base
+          include ArticleJSON::Export::Common::HTML::Elements::Base
+          include ArticleJSON::Export::Common::HTML::Elements::Text
+
+          def export
```

Comment:
I think it'd be worth adding a bit of documentation here, why are we using super and calling to_s on it?